rviz_default_plugins/test/rviz_default_plugins/displays/pointcloud/point_cloud_transformers/axis_color_pc_transformer_test.cpp

```diff
@@ -36,7 +36,9 @@
 #include "rviz_common/properties/property.hpp"
 #include "../message_creators.hpp"
 
-#include "../src/rviz_default_plugins/point_cloud_transformers/axis_color_pc_transformer.hpp"
+// *INDENT-OFF*
+#include "../../../../../src/rviz_default_plugins/displays/pointcloud/transformers/axis_color_pc_transformer.hpp"
```

Comment:
only a minor comment. But looking at the filepath include. When it's so deeply nested included, doesn't it make sense to make `axis_color_pc_transformer.hpp` a public header?

Reply:
This is a test, and the header should only be public if the header is used outside of the package. So, no.

Reply:
true. Didn't realize it was a test. Sorry for the noise.

Reply:
Yeah, otherwise (had this been a source file for a library or executable) I would either collocate the header with the source file (if it were private) or I would have made it a public header and put it in include.

👍